### PR TITLE
MatchColumns, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Source has been updated from the first release, Makefile is included, so need to look that over, aside from that altering the recipe to use a srcGitRev should be fairly simple ...

**Comment by fbrosson** [comment]
@Begasus, I think it would be nice to rename the directory to "matching-columns" (but leave the name of the recipe unchanged.)

File: haiku-games/matchingcomulns/matching_columns-0.1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
```
Comment (fbrosson) [inline haiku-games/matchingcomulns/matching_columns-0.1.0.recipe]: Suggestion:
```
defineDebugInfoPackage matching_columns \
	$appsDir/MatchColumns
```
(BTW, the game works fine on x86_gcc2)

File: haiku-games/matchingcomulns/matching_columns-0.1.0.recipe [hunk 2]
```diff
@@ -0,0 +1,42 @@
+SUMMARY="A matching game"
+DESCRIPTION="MatchColumns, a fun matching game."
```
Comment (alaviss) [inline haiku-games/matchingcomulns/matching_columns-0.1.0.recipe]: This one should serve as a better `DESCRIPTION`

From [spiceweasel/matching-columns](https://github.com/spiceweasel/matching-columns):

> A game for the Haiku OS where users score points by matching the columns before breaking them.

File: haiku-games/matchingcomulns/matching_columns-0.1.0.recipe [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+HOMEPAGE="https://github.com/spiceweasel/matching-columns"
+COPYRIGHT="2017 Scott Jacovidis"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="eac403db4a639b0893f47738896c60cb37f569c1"
+SOURCE_URI="https://github.com/spiceweasel/matching-columns/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="a042f9bc916a10332bd1475bd55af64cd71c28664afbfc333a4b57d558ad85ff"
+SOURCE_FILENAME="matching-columns-$srcGitRev.tar.gz"
```
Comment (alaviss) [inline haiku-games/matchingcomulns/matching_columns-0.1.0.recipe]: You should update this according to the [Guidelines][0]. The prefered format for this kind of package is `matchingcolumns-$portVersion-$srcGitRev.tar.gz`

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name

File: haiku-games/matchingcomulns/matching_columns-0.1.0.recipe (hunk 1 above)
Comment (Begasus) [reply haiku-games/matchingcomulns/matching_columns-0.1.0.recipe]: Already found out :) (missing alloc.h for gcc5)

File: haiku-games/matchingcomulns/matching_columns-0.1.0.recipe (hunk 2 above)
Comment (fbrosson) [reply haiku-games/matchingcomulns/matching_columns-0.1.0.recipe]: Yep, that's a nice description, but I'd change a small thing: `for the Haiku OS` -> `for Haiku`

@alaviss, I think the Haiku community prefers to call the system just "Haiku",  without the "OS".
(I also think it sounds better.)
We even say that "Haiku was inspired by BeOS".
The fact that the website address is haiku-os.org is an "accident": when the name was chosen, haiku.org was already taken and I guess their owners did not seem to be willing to give it for free (or for a friendly price :)

**Comment by pulkomandy** [comment]
On Sun, Jan 14, 2018 at 12:57:37AM -0800, fbrosson wrote:
> We even say that "Haiku was inspired by BeOS".

While we are at "getting names rights", the correct phrasing is
"inspired by the BeOS" (a shorthand for "the Be Operating System", which
never really got a proper name ;))

**Comment by Begasus** [comment]
Highscores are not saved (don't find an error when trying to run it in Terminal)

**Pull request merged**